### PR TITLE
Improved the prosumers dashboard and removed managers dashboard

### PR DESCRIPTION
**Pull request by Aleman778**

### CODE REVIEW

File: web-server/public/js/climate/wind-speed-updater.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ $(window).on("unload", function() {
  */
 async function updateWindSpeedDataField() {
     try {
-        const response = await fetch('http://localhost:3000/api/wind');
+        const response = await fetch('prosumer/climate/wind');
```
Comment (Blinningjr) [inline web-server/public/js/climate/wind-speed-updater.js]:
```suggestion
        const response = await fetch('/prosumer/climate/wind');
```

File: web-server/routes/prosumer.js [hunk 2]
```diff
@@ -210,6 +210,19 @@ router.post('/market/price',
             marketController.getPrice);
 
 
+
+/**
+ * GET request /prosumer/climate/wind for getting wind speed
+ */
+router.get('/climate/wind', prosumerController.getWindSpeed);
```
Comment (Blinningjr) [inline web-server/routes/prosumer.js]:
```suggestion
router.get('/climate/wind', auth.verify, prosumerController.getWindSpeed);
```

File: web-server/routes/prosumer.js [hunk 3]
```diff
@@ -210,6 +210,19 @@ router.post('/market/price',
...
+
+
+/**
+ * GET request /prosumer/climate/wind for getting wind speed
+ */
+router.get('/climate/wind/history', prosumerController.getWindSpeedHistory);
```
Comment (Blinningjr) [inline web-server/routes/prosumer.js]:
```suggestion
router.get('/climate/wind/history', auth.verify, prosumerController.getWindSpeedHistory);
```

File: web-server/routes/prosumer.js (hunk 2 above)
Comment (Aleman778) [reply web-server/routes/prosumer.js]: Wind is public information there is no need to authenticate

**Pull request merged**